<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorDetailsDialog</class>
 <widget class="QDialog" name="DoctorDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="photoLabel">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #eee;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="nameLabel">
               <property name="font">
                <font>
                 <family>Ubuntu Condensed</family>
                 <weight>75</weight>
                 <italic>true</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>医生姓名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="titleLabel">
               <property name="text">
                <string>职称</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="departmentLabel">
               <property name="text">
                <string>所属科室</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="genderLabel">
               <property name="text">
                <string>性别</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="bioTextBrowser"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
